MIGRATION.md

```diff
@@ -0,0 +1,109 @@
+# Eslint Migrations Guide
+
+## Migration from eslint 8.x.x to 9.x.x
+
+The following steps are recommended when migrating from an existing eslint 8.x.x configuration using .eslint.rc configuration file to an eslint 9.x.x setup using `@iobroker/eslint-config`.
+The steps are valid for classic javascript and typescritp repositories. If you are using esm modules or react the eslint configuration needs to include some more modules. Please see README.md for details.
+
+- [ ] Clone your repository to a local workspace
+- [ ] Verify that current (old) eslint configuration is working
+
+  Execute `npm run lint` and check for errors. Consider fixing all existing errors before starting the migration.
+      
+- [ ] remove packages no longer needed
+
+      npm uninstall eslint-config-prettier
+      npm uninstall eslint-plugin-prettier
+      npm uninstall prettier
+
+- [ ] update eslint to current version
+
+      npm i eslint@latest --save-dev
+
+- [ ] install iobroker standard rules
+
+      npm i @iobroker/eslint-config --save-dev
+
+- [ ] remove old configuration files
+
+      .eslintignore
+      .eslintrc.json
+      .prettierignore
+      .prettierrc.js
+      
+- [ ] create new configuration file 'eslint.config.mjs' for eslint
```

Comment:
create new configuration file __in the root directory__

Reply:
done